src/_kw

```diff
@@ -285,6 +285,7 @@ _kw_init()
     '(-a --arch)'{-a,--arch}'[set the variable arch of kw]: : ' \
     '(-r --remote)'{-r,--remote}'[set the remote user, ip and port]: : ' \
     '(-t --target)'{-t,--target}'[set the default_deploy_target config of kw between local or remote]: :(local remote)' \
+    '(--guided-setup)--guided-setup[step-by-step of the basic setup]' \
```

Comment:
In case we want to make `--guided-setup` mutually exclusive to any other option, do the following:
```
diff --git a/src/_kw b/src/_kw
index 432804b..d530110 100644
--- a/src/_kw
+++ b/src/_kw
@@ -282,10 +282,10 @@ _kw_init()
 {
   _arguments : \
     '(--verbose)--verbose[enable verbose mode]' \
-    '(-a --arch)'{-a,--arch}'[set the variable arch of kw]: : ' \
-    '(-r --remote)'{-r,--remote}'[set the remote user, ip and port]: : ' \
-    '(-t --target)'{-t,--target}'[set the default_deploy_target config of kw between local or remote]: :(local remote)' \
-    '(--guided-setup)--guided-setup[step-by-step of the basic setup]' \
+    '(-a --arch --guided-setup)'{-a,--arch}'[set the variable arch of kw]: : ' \
+    '(-r --remote --guided-setup)'{-r,--remote}'[set the remote user, ip and port]: : ' \
+    '(-t --target --guided-setup)'{-t,--target}'[set the default_deploy_target config of kw between local or remote]: :(local remote)' \
+    '(-a --arch -r --remote -t --target --guided-setup)--guided-setup[step-by-step of the basic setup]' \
     '--template=-[pass no arguments to list all available templates or pick one by passing its name]: : '
 }
```